Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/WheelWithBypassDamper.mo

```diff
@@ -0,0 +1,296 @@
+    final dpDamper_nominal=dp1_nominal)
+    "Damper in the supply air stream"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={-32,36})));
+  Buildings.Fluid.Actuators.Dampers.Exponential damExh(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal,
+    final dpDamper_nominal=dp2_nominal)
+    "Damper in the exhaust air stream"
+    annotation (Placement(transformation(
+        extent={{10,10},{-10,-10}},
+        rotation=-90,
+        origin={60,-40})));
+  Buildings.Fluid.Actuators.Dampers.Exponential bypDamExh(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal,
+    final dpDamper_nominal=dp2_nominal)
+    "Bypass damper in the exhaust air stream"
+    annotation (Placement(transformation(extent={{-20,-70},{-40,-50}})));
+  Modelica.Blocks.Sources.RealExpression TSup(
+      final y=Medium1.temperature(
+      Medium1.setState_phX(
+        p=port_a1.p,
+        h=inStream(port_a1.h_outflow),
+        X=inStream(port_a1.Xi_outflow))))
+    "Temperature of the supply air"
+    annotation (Placement(transformation(extent={{-160,-30},{-140,-10}})));
+  Modelica.Blocks.Sources.RealExpression TExh(
+      final y=Medium2.temperature(
+      Medium2.setState_phX(
+        p=port_a2.p,
+        h=inStream(port_a2.h_outflow),
+        X=inStream(port_a2.Xi_outflow))))
+    "Temperature of the exhaust air"
+    annotation (Placement(transformation(extent={{-160,-50},{-140,-30}})));
+  Modelica.Blocks.Sources.RealExpression PEle(y=P_nominal)
+    "Calculates the electric power consumption"
+    annotation (Placement(transformation(extent={{70,-10},{90,10}})));
+  Modelica.Fluid.Interfaces.FluidPort_a port_a1(
+    redeclare final package Medium = Medium1)
+    "Fluid connector a1 of the supply air"
+    annotation (Placement(transformation(extent={{-190,70},{-170,90}}),
+        iconTransformation(extent={{-110,50},{-90,70}})));
+  Modelica.Fluid.Interfaces.FluidPort_b port_b2(
+    redeclare final package Medium = Medium2)
+    "Fluid connector b2 of the supply air"
+    annotation (Placement(transformation(extent={{-170,-70},{-190,-50}}),
+        iconTransformation(extent={{-90,-70},{-110,-50}})));
+  Modelica.Fluid.Interfaces.FluidPort_b port_b1(
+    redeclare final package Medium = Medium1)
+    "Fluid connector b1 of the exhaust air"
+    annotation (Placement(transformation(extent={{110,70},{90,90}}),
+        iconTransformation(extent={{110,50},{90,70}})));
+  Modelica.Fluid.Interfaces.FluidPort_a port_a2(
+    redeclare final package Medium = Medium2)
+    "Fluid connector a2 of the exhaust air"
+    annotation (Placement(transformation(extent={{90,-70},{110,-50}})));
+  Modelica.Blocks.Sources.Constant uni(final k=1)
+    "Unity signal"
+    annotation (Placement(transformation(extent={{-154,32},{-140,46}})));
+  Modelica.Blocks.Math.Add sub(final k1=-1, final k2=+1)
```

Comment:
It might be easier to use the `Buildings.Controls.OBC.CDL.Reals.Subtract`. And change the comment to be as `Difference of the two inputs`.